src/NodaTime/CalendarSystem.cs

```diff
@@ -189,6 +191,7 @@ public static CalendarSystem ForId(string id)
         /// this implementation align exactly with the BCL implementation.
         /// </remarks>
         /// <returns>A Persian calendar system.</returns>
+        [Obsolete("Use the PersianSimple property for compatibility with 2.0.")]
```

Comment:
You're not obsoleting the others because there's not always an alternative?

(If so, I'm not sure about that: they are obsolete, even if they can't be directly replaced.)